src/casp/services/api/routers/cell_operations_router.py

```diff
@@ -11,13 +12,11 @@
 cellarium_general_service = services.CellariumGeneralService()
 
 
-@cell_operations_router.post(
-    "/annotate", response_model=t.Union[t.List[schemas.QueryCellDevDetails], t.List[schemas.QueryCell]]
-)
+@cell_operations_router.post("/annotate", response_model=schemas.QueryAnnotationCellTypeCountType)
 async def annotate(
     file: UploadFile = File(),
     model_name: str = Form(),
-    include_dev_metadata: bool = Form(),
+    include_dev_metadata: t.Optional[bool] = Form(default=None),
```

Comment:
`annotate` -> `annotate_cell_type_count_strategy` (no implied default)
`include_dev_metadata` -> `include_extended_metadata` (here and elsewhere)